elements/lisk-p2p/test/unit/peer/base.ts

```diff
@@ -338,8 +340,8 @@ describe('peer/base', () => {
 				.spyOn(defaultPeer as any, 'request')
 				.mockResolvedValue({
 					data: {
-						peers: [],
 						success: true,
+						peers: [],
```

Comment:
Does the order of object property matter here

Reply:
no, it doesn't

Reply:
Ok, I saw that you changed it so I wondered if there was a reason.

Reply:
ah yeah, I changed it because I thought it was the reason for failing test :(